Adding links to doc for device twins.

WilliamBerryiii opened the pull request:
It would additionally be nice to include code samples here for performing desired property updates.

azurecla commented:
Hi __@tysonn__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (tysonn). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

WilliamBerryiii committed:
Adding links to doc for device twins.

It would additionally be nice to include code samples here for performing desired property updates.
